### ISSUE TITLE
Certain folder won't show up.

### ISSUE THREAD

**Issue by mariacheline** [Microsoft/vscode#33780]
- VSCode Version: Code 1.15.1 (41abd21afdf7424c89319ee7cb0445cc6f376959, 2017-08-16T18:07:25.676Z)
- OS Version: Windows_NT x64 10.0.15063
- Extensions: 

Extension|Author (truncated)|Version
---|---|---
unity3d-pack|dim|0.7.1
docomment|k--|0.0.17
csharpfixformat|Leo|0.0.32
unity-snippets-modified|lio|0.3.2
csharp|ms-|1.12.1
shader|sle|0.2.3
unity-tools|Tob|0.1.9
unity-debug|Uni|1.2.1
unity-snippets|Ycl|0.1.2
mono-snippets|zra|0.0.2

(1 theme extensions excluded)

---

Basically a certain folder won't show up on editor.

Steps to Reproduce:

1. Open folder either via vscode or file explorer

Reproduces without or without extensions.

**Comment by sandy081** [comment]
Duplicate of #32962

Please reopen if it is not and provide more specific steps
